zokrates_book/src/language/imports.md

```diff
@@ -13,6 +13,12 @@ from "./path/to/my/module" import MySymbol
 // `MySymbol` is now in scope.
 ```
 
+To import multiple modules with a single import statement, separate the module names with commas:
```

Comment:
```suggestion
To import multiple symbols with a single import statement, separate the symbol names with commas:
```

Reply:
Oh, how did I not notice this :facepalm: Nice catch!